addons/test_website/static/tests/tours/restricted_editor.js

```diff
@@ -7,11 +7,15 @@ import {
 
 const checkNoTranslate = {
     content: "Check there is no translate button",
-    trigger: ".o_menu_systray:not(:has(.o_translate_website_container)):contains(edit)",
+    trigger: ".o_menu_systray:not(:has(.o-dropdown-toggle-custo)):contains(edit)",
+    async run() {
+        // TODO: check if we can avoid this timeout
+        await new Promise((r) => setTimeout(r, 900));
```

Comment:
Yes, it's never a good idea to delay. Can you explain exactly why you need this? Do we have a loading screen where you can check that it's no longer there?